antora-lunr-ui/partials/header-content.hbs

```diff
@@ -1,14 +1,16 @@
 <header class="header" role="banner">
   <nav class="navbar">
-    <div class="navbar-brand">
-      <a class="navbar-item" href="{{or site.url (or siteRootUrl siteRootPath)}}">{{site.title}}</a>
-
-      <button class="navbar-burger" data-target="topbar-nav">
-        <span></span>
-        <span></span>
-        <span></span>
-      </button>
-    </div>
+        <div class="navbar-brand">
+          <div class="navbar-item">
+            <button class="navbar-burger" data-target="topbar-nav">
+              <span></span>
+              <span></span>
+              <span></span>
+            </button>
+            <img src="{{uiRootPath}}/img/logo.png" class="navbar-logo" alt="App Studio logo">
+            <a href="{{site.url}}/">{{site.title}}</a>
+          </div>
+        </div>
```

Comment:
It's up to you, but if this was my code I would move the unrelated logo changes into their own separate commit and give them a proper commit message.